### ISSUE TITLE
Update preview video

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#311]
### Terms

- [X] I have searched [open and closed issues](https://github.com/activist-org/activist/issues?q=is%3Aissue)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Issue

The video that we have on the [readme](https://github.com/activist-org/activist) is super out of date and reflects behaviors that would not be included in activist such as collecting phone numbers and other form entries. This issue has been known for some time, but focus has been on other places. It would be good to update this with more up to date versions from the designs.

**Comment by andrewtavis** [comment]
- Note that this is blocked by the revamp of the colors/designs in #215 :) Once that's done I'll redo some previews.

**Comment by andrewtavis** [comment]
Considering this right now, it might be better if we update this with a video of the actual platform rather than spending the time to make one of the designs :)

**Comment by willsaliba** [comment]
Hi, I would be happy to remake this video. 

Would you be able to provide exact details on what you'd like to see in the new video or alternatively what you'd like to see different about the old one.

**Comment by andrewtavis** [comment]
Hey @willsaliba 👋 Thanks for your interest in this issue. Blocking it for now as the goal would be that we'd use the platform for the new video, and there's still a good amount of work left to be done for that to be possible. We can discuss this a bit when the full navigation is complete! Saw you wrote into another issue, so will head over there now 😊
